test/models/criterium_test.rb

```diff
@@ -10,10 +10,9 @@ class CriteriumTest < ActiveSupport::TestCase
   test "should not save with both 'none of the above' other options" do
```

Comment:
And errors here:

<img width="850" alt="Screenshot 2022-03-28 at 17 47 32" src="https://user-images.githubusercontent.com/40244233/160447396-105b0da8-5a71-43a7-99b5-651a35d5f1fb.png">

```
.F

Failure:
CriteriumTest#test_should_not_save_with_both_'none_of_the_above'_other_options [/Users/yndajas/dev/UNEP-WCMC/action-agenda/test/models/criterium_test.rb:13]:
Expected false to be truthy.


bin/rails test test/models/criterium_test.rb:10

F

Failure:
CriteriumTest#test_should_be_invalid_for_commitment_if_it_has_negative_values [/Users/yndajas/dev/UNEP-WCMC/action-agenda/test/models/criterium_test.rb:25]:
Expected true to be nil or false


bin/rails test test/models/criterium_test.rb:20

F

Failure:
CriteriumTest#test_should_give_the_correct_failure_messages [/Users/yndajas/dev/UNEP-WCMC/action-agenda/test/models/criterium_test.rb:34]:
--- expected
+++ actual
@@ -1 +1 @@
-[]
+["Your commitment is being made by a government"]
```

Reply:
not getting errors on this branch now, but happening on other branches. maybe needs db reset?

Changed column name as requested